Makefile

```diff
@@ -40,3 +43,7 @@ certs:
 	@mkcert -key-file ./docker/traefik/certs/key.pem -cert-file ./docker/traefik/certs/cert.pem \
 		'purrfect-ui.localhost' \
 		'purrfect-ui.traefik.localhost'
+
+help: ## Outputs this help screen
+	@grep -E '(^[a-zA-Z0-9\./_-]+:.*?##.*$$)|(^##)' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}{printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}' | sed -e 's/\[32m##/[33m/'
+.DEFAULT_GOAL := help
```

Comment:
I understand the pre-commit, but I don't understand this one. Could you explain me please ?

Reply:
This is a default target which lists the available targets when you call `make` without specifying a target.

Reply:
Maybe replace `ci:` content by pre-commit one ? Also, nothing but pnpm and help will be shown cause of the missing comments ?

Reply:
It would be a bad idea, as format:check --write can actually apply changes to the code.
I'll add the missing docs.